test/distrib/bazel/test_single_bazel_version.sh

```diff
@@ -67,6 +67,9 @@ EXCLUDED_TARGETS=(
 
   # Also exclude the artifact_gen tooling, which again contains some bazel hackery
   "-//tools/artifact_gen/..."
+
+  # Also exclude the codegen tooling, which again contains some bazel hackery
+  "-//tools/codegen/..."
```

Comment:
Instead of excluding everything, would you exclude relevant targets only? I think most of codegen targets still need to be covered here.

Reply:
Sure, done.